fastestimator/backend/random_mix_patch.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 The FastEstimator Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from typing import List, TypeVar, Union
+
+import tensorflow as tf
+import torch
+
+Tensor = TypeVar('Tensor', tf.Tensor, torch.Tensor)
+
+
+def random_mix_patch(tensor: Tensor, lam: Tensor, uniform_sample: Tensor) -> Tensor:
+    """Randomly cut the patches from input images.
+
+    Args:
+        tensor: The input value.
+        lam: Generated mixed sample.
```

Comment:
scribe what these values are used for. if this function could be invoked using a number not drawn from a  uniform distribution, then the arg name shouldn't be 'uniform_sample'

also, doc string lines should all end with '.'